oracle/export_test_vectors/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "export_test_vectors"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+ark-ff = { version = "0.3.0", features = [ "parallel", "asm" ] }
+num-bigint = { version = "0.4.0", optional = true }
+serde_json = { version = "1.0", optional = true }
+hex = { version = "0.4", optional = true }
+ark-serialize = { version = "0.3.0", optional = true }
+rand = "0.8.0"
+serde = { version = "1.0", features = ["derive"] }
+serde_with = "1.10.0"
+
+mina-curves = { path = "../../curves" }
+oracle = { path = "../../oracle" }
+
+[features]
+default = ["num-bigint", "serde_json", "hex", "ark-serialize"]
```

Comment:
I don't think we need any features here, so you can just all the `optional = true`